modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1737,7 +1738,10 @@ private CompletableFuture<List<Assignments>> getOrCreateAssignments(
     protected MvTableStorage createTableStorage(CatalogTableDescriptor tableDescriptor, CatalogZoneDescriptor zoneDescriptor) {
         StorageEngine engine = dataStorageMgr.engineByStorageProfile(tableDescriptor.storageProfile());
 
-        assert engine != null : "tableId=" + tableDescriptor.id() + ", storageProfile=" + tableDescriptor.storageProfile();
+        if (engine == null) {
```

Comment:
Does everything else work?

Reply:
Could you please elaborate what `everything else` means here? At least, tests are green.

Reply:
I mean, that's an arbitrary question. I'm a little shocked that it's one of just a few places that you had to modify